code/src/components/PreviousPositions.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+export const PreviousPositions = () => {
+
+const history = useSelector(store => store.gameReducer.history);  
+const historyFiltered = history.filter((item,index) => index !== 0).reverse(); 
+
+console.log("History array lenght: ",history.length);
+  return (
+    <div className="gameboard-history">
```

Comment:
This div doesn't contain any styling, so here you could use the react-fragment instead 👍